std/http/server.ts

```diff
@@ -247,10 +275,10 @@ export type HTTPOptions = Omit<Deno.ListenOptions, "transport">;
  *       req.respond({ body });
  *     }
  */
+
 export function serve(addr: string | HTTPOptions): Server {
```

Comment:
Please don't add a newline here. I think it might break the JSDoc.

Reply:
Fixed